Tests/test_image_paste.py

```diff
@@ -0,0 +1,112 @@
+from helper import unittest, PillowTestCase, hopper
+
+from PIL import Image
+
+
+class TestImagingPaste(PillowTestCase):
+    masks = {}
+
+    def mask_1(self, size):
+        mask = Image.new('1', size)
+        px = mask.load()
+        for y in range(mask.height):
+            for x in range(mask.width):
+                px[y, x] = (x + y) % 2
+        return mask
+
+    def mask_L(self, size):
+        mask = Image.new('L', size)
+        px = mask.load()
+        for y in range(mask.height):
+            for x in range(mask.width):
+                px[y, x] = (x + y) % 255
+        return mask
+
+    def mask_RGBA(self, size):
+        mask = Image.new('RGB', size, 'red').split()
+        return Image.merge('RGBA', mask + (self.mask_L(size),))
+
+    def mask_RGBa(self, size):
+        mask = Image.new('RGB', size, 'red').split()
+        return Image.merge('RGBa', mask + (self.mask_L(size),))
+
+    def test_image_solid(self):
+        for mode in ('L', 'RGB'):
+            im = Image.new(mode, (200, 200), 'red')
+            im2 = hopper(mode)
+
+            im.paste(im2, (0, 0))
+
+            im = im.crop((0, 0) + im2.size)
+            self.assert_image_equal(im, im2)
+
+    def test_image_mask_1(self):
```

Comment:
Other than checking these run without exception, is there something that can/should be explicitly asserted in this and the following tests?